src/Compilers/Core/Portable/Emit/EmitOptions.cs

```diff
@@ -104,6 +105,18 @@ public sealed class EmitOptions : IEquatable<EmitOptions>
         /// </summary>
         public string? RuntimeMetadataVersion { get; private set; }
 
+        /// <summary>
+        /// The default source encoding used to parse source before producing a compilation. Only embedded in
```

Comment:
>Only [](start = 93, length = 4)

"Only embedded in the pdb ..." needs context in order to be understood. I'd say "The value, if specified, is stored in the emitted PDB in order to allow recreating the original compilation."